### PR TITLE
add models directory

### PR DESCRIPTION
**Pull request by ryanjgani**

### CODE REVIEW

**Comment by iamdejan** [comment]
Next time for pull request

Please describe:
1) What are the changes
2) Where to start reviewing (the sequence of the files)

File: models/notifications.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const notificationSchema = new Schema({
+    new_fundraiser: {
+        type: Schema.Types.ObjectId,
+        ref: "User", //refer to the User model
+    },
+    new_withdrawal_request: {
+        user_id: {
+            type: Schema.Types.ObjectId,
+            ref: "User", //refer to the User model
```
Comment (iamdejan) [inline models/notifications.js:12]: example comment

File: models/donations.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const donationSchema = new Schema({
+    donator_id: {
+        type: Schema.Types.ObjectId,
+        ref: "Donation", //refer to the Donation model
```
Comment (iamdejan) [inline models/donations.js:7]: how to refer programmatically?

File: models/donations.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    },
+    program_id: {
+        type: Schema.Types.ObjectId,
+        ref: "Program", //refer to the Program model
```
Comment (iamdejan) [inline models/donations.js:11]: how to refer this in code?

File: models/notifications.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        },
+        amount: {
+            type: Number,
+        },
+    },
+    new_program: {
```
Comment (cholazzzb) [inline models/notifications.js:18]: why not new_program_id?

**Review by cholazzzb — APPROVED** [review]

**Pull request merged**